src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -304,6 +308,7 @@ def __str__(self):
         app_insights_telemetry_channel_resource_id='https://dc.applicationinsights.us/v2/track'),
     suffixes=CloudSuffixes(
         storage_endpoint='core.usgovcloudapi.net',
+        storage_sync_endpoint='afs.azure.net',
```

Comment:
The endpoint is the same for AzureUsGovernment and AzureCloud?

Reply:
Thanks @qianwens for pointing it out. My negligence, updated.